packages/components/src/theme/ComposerTheme/Components/Button.ts

```diff
@@ -51,29 +51,18 @@ const generateAltButton = (color: string) => {
     backgroundColor: 'background-action-disabled',
     opacity: 1,
     color: transparentize(`text-link-${color}-default`, 0.6),
-    borderColor: transparentize(`text-link-${color}-default`, 0.6),
+    outlineColor: `${color}-disabled`,
   };
 
-  let loadingStyles = {
-    borderColor: `${color}-default`,
-    borderBottomColor: transparentize(`${color}-default`, 0.3),
-    borderLeftColor: transparentize(`${color}-default`, 0.3),
-  };
-
-  if (color === 'secondary') {
-    loadingStyles = {
-      borderColor: 'primary-default',
-      borderBottomColor: transparentize('primary-default', 0.3),
-      borderLeftColor: transparentize('primary-default', 0.3),
-    };
-  }
```

Comment:
I removed these lines since they do the same, basically, it sets `primary` styles even when it is a secondary color.